pages/app/login.js

```diff
@@ -11,6 +13,22 @@ const description =
 export default function Login() {
   const [loading, setLoading] = useState(false);
 
+  //Get error message added by next/auth in URL.
+  const { error } = useRouter().query;
+  useEffect(() => {
+    if (error) {
+      //Pop latest if there are multiple error query in URL
+      let errorMessage;
+      if (Array.isArray(error)) {
+        errorMessage = error.pop(error);
+      } else {
+        errorMessage = error;
+      }
```

Comment:
Could probably clean this up a bit with some ternary operators 🙂 
```suggestion
      const errorMessage = Array.isArray(error) ? error.pop(error) : error;
```